### DISCUSSION TITLE
RawInput key for LeftAlt & RightAlt?

### DISCUSSION THREAD

**Discussion by jeffhgit** [Elringus/UnityRawInput#16] [Q&A] [answered]
I'm trying to assign the Alt keys, but there is no definition for them:

![image](https://user-images.githubusercontent.com/87553869/179873639-6e821ca1-5fea-48b8-9090-91db4d6943d8.png)

I've searched RawKey.cs, no matches:

![image](https://user-images.githubusercontent.com/87553869/179873604-153aced7-aa74-431d-a4fc-79a8d9f1a918.png)

Are the Alt keys called something else, or are they not coded? If so, how can I add them? Thank you.

**Comment by jeffhgit** [comment, marked as answer]
Figured it out:
LeftAlt = LeftMenu
RightAlt = RightMenu

**Comment by elringus** [comment]
Here is the codes table: https://docs.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes
